base-template/about-base-template.md

```diff
@@ -1,8 +1,21 @@
 # About Base Template
 
+{Template author tip:
+
+* This about-base-template captures the structure and common elements that all downstream about-xxx-templates should follow.
+* It provides guidance for both template authors as well as document authors.
+* “Template author and template reviewer tips” should be removed from the final about-{doctype}-template.md. Other tips should remain.
+* Tweak other tips to align with the doctype you are writing for.
+
```

Comment:
I feel like I'm missing the forest for the trees on this document (which is how I felt about the previous version as well). Perhaps what is missing is an overview statement at the beginning that orients me about the overall purpose and use of this document. If I understand this correctly (again, really not sure), the purpose of this template is to guide future template creators on The Good Docs Project. With that in mind, here's a very, very tentative paragraph that you are welcome to change to your heart's content:

```
The base template is for use in The Good Docs Project. Before using this template, see <link to our Contributing Guide, maybe calling out the sections that give and overview define the process for submitting draft templates>. This template outlines the stylistic conventions and overall structure for creating a new Good Docs Project template.
```

Another comment: A good template guides the user in the process of making decisions about how to adapt and use the template for their specific context. I feel like an About template should provide this kind of direction. I'm not sure how to represent that here, but that's what I feel like I'm missing. Sorry that's very vague. I want to think about it some more.

Last comment: The real test will be to see if someone could take this base template and use it effectively to create a new template. Let's consider piloting that kind of test and keeping what Erin calls "a friction log" of where the template helps and hinders.